README.md

```diff
@@ -72,3 +72,18 @@ There are subtle differences between the cross-account and same-account situatio
 To begin with, the destination bucket needs a policy that allows the source account to write to replicate to it. Because we are adding a bucket policy, you will also then need to add additional permissions for users in the destination bucket.
 
 Similarly, the KMS key in the destination account needs to allow access from the source account. By only allowing `kms:Encrypt` action, the access permission does not need to be more complex.
+
+## License
+Copyright 2018 Leap Beyond Analytics
```

Comment:
Shouldn't this be 2020? Or is it something we can safely leave at a past date?

Reply:
2018 was when we published it.